suite-native/graph/src/components/Graph.tsx

```diff
@@ -89,8 +92,10 @@ export const Graph = <TGraphPoint extends FiatGraphPoint>({
         applyStyle,
         utils: { colors },
     } = useNativeStyles();
+    const { translate } = useTranslate();
 
     const isPointsEmpty = points.length <= 1;
+    const isDiscoveryActive = useSelector(selectIsDeviceDiscoveryActive);
```

Comment:
Please pass this as prop to this component, I want this component to be "relatively" pure, also it will show loader also on account details of already loaded coins now etc.

Reply:
This selector removed for the same reason as stated in previous comment. [da8a59d](https://github.com/trezor/trezor-suite/pull/10030/commits/da8a59d889d710e58aee11beab14f0dcc0c0cdf1)